main.js

```diff
@@ -65,22 +63,18 @@ function myTank() {
     // Updates the autopilot with the current state
     autopilot.update(state, control);
 
-    if (!firstEnemyFound && state.radio.inbox.length) {
+    if (!state.radar.enemy && state.radio.inbox.length) {
```

Comment:
If they are switching targets too much, you could try:
```
if (!(enemy && enemyAge < ENEMY_MAX_AGE) && !state.radar.enemy && state.radio.inbox.length)
```

But what you have is good too